core/server/master/src/main/java/alluxio/master/file/options/CheckConsistencyOptions.java

```diff
@@ -22,21 +23,71 @@
  */
 @NotThreadSafe
 public final class CheckConsistencyOptions {
+  private CommonOptions mCommonOptions;
+
   /**
-   * @return the default {@link CompleteFileOptions}
+   * @return the default {@link CheckConsistencyOptions}
    */
-  @SuppressFBWarnings("ISC_INSTANTIATE_STATIC_CLASS")
   public static CheckConsistencyOptions defaults() {
     return new CheckConsistencyOptions();
   }
 
+  private CheckConsistencyOptions() {
+    mCommonOptions = CommonOptions.defaults();
+  }
+
   /**
    * Constructs an instance of {@link CheckConsistencyOptions} from
    * {@link alluxio.thrift.CheckConsistencyTOptions}.
    *
    * @param options the {@link alluxio.thrift.CheckConsistencyTOptions} to use
    */
-  public CheckConsistencyOptions(CheckConsistencyTOptions options) {}
+  public CheckConsistencyOptions(CheckConsistencyTOptions options) {
+    this();
+    if (options != null) {
+      if (options.isSetCommonOptions()) {
```

Comment:
nit: could use one `if` with `&&`

Reply:
I think wanted to keep it this way, in case we add new options to this class.